nupm/install.nu

```diff
@@ -23,107 +23,161 @@ def throw-error [
     }
 }
 
-def open-package-file [path: path] {
-    let package_file = $path | path join "package.nuon"
+def open-package-file [dir: path] {
+    let package_file = $dir | path join "package.nuon"
 
     if not ($package_file | path exists) {
-        throw-error $"package_file_not_found(ansi reset):\nno 'package.nuon' found in ($path)"
+        throw-error (
+            [
+                $"package_file_not_found(ansi reset):"
+                $"no 'package.nuon' found in ($dir)"
+            ]
+            | str join (char nl))
     }
 
     let package = open $package_file
 
     log debug "checking package file for missing required keys"
-    let required_keys = [$. $.name $.version $.description $.license $.type]
-    let missing_keys = $required_keys | where {|key| ($package | get -i $key) == null}
+    let required_keys = [$. $.name $.version $.type]
+    let missing_keys = $required_keys
+        | where {|key| ($package | get -i $key) == null}
     if not ($missing_keys | is-empty) {
-        throw-error $"invalid_package_file(ansi reset):\n($package_file) is missing the following required keys: ($missing_keys | str join ', ')"
+        throw-error (
+            [
+                $"invalid_package_file(ansi reset):"
+                ($"($package_file) is missing the following required keys:"
+                    + $" ($missing_keys | str join ', ')")
+            ]
+            | str join)
```

Comment:
i think this should `str join` on `char nl` to preserve the newline in the original error, right?

Reply:
done in https://github.com/nushell/nupm/pull/18/commits/983a645c1d999399f64f00c385963e163cd19193